docs/source/index.rst

```diff
@@ -1,132 +1,89 @@
-.. _Modbus-Driver:
+.. _DNP3-Driver:
 
-=============
-Modbus Driver
-=============
+===========
+DNP3 Driver
+===========
 
+VOLTTRON's DNP3 driver enables the use of `DNP3 <https://en.wikipedia.org/wiki/DNP3>`_ (Distributed Network Protocol)
+communications, reading and writing points via a DNP3 Outstation.
 
-
-VOLTTRON's modbus driver supports the Modbus over TCP/IP protocol only. For Modbus RTU support, see VOLTTRON's
-`Modbus-TK driver <Modbus-TK-Driver>`.
-
-`About Modbus protocol <https://en.wikipedia.org/wiki/Modbus>`_
-
-
-.. _Modbus-Config:
-
-Modbus Driver Configuration
-===========================
+In order to use a DNP3 driver to read and write point data, a server component (i.e., Outstation) must also
+be configured and running.
 
 Requirements
-------------
-The Modbus driver requires the pymodbus package. This package can be installed in an activated environment with:
+============
 
-.. code-block:: bash
-
-    pip install pymodbus
-
-Alternatively this requirement can be installed using :ref:`bootstrap.py <Platform-Installation>` with the ``--drivers``
-option:
+The DNP3 driver requires the `dnp3-python <https://github.com/VOLTTRON/dnp3-python>`_ package, a wrapper on Pydnp3 package.
+This package can be installed in an activated environment with:
 
 .. code-block:: bash
 
-    python3 bootstrap.py --drivers
+    pip install dnp3-python
 
 
 Driver Configuration
---------------------
+====================
+
+There is one argument for the "driver_config" section of the DNP3 driver configuration file:
```

Comment:
You seem to show 5 arguments:   master_ip, outstation_ip, master_id, outstation_id, and port. While the names are probably fairly self-explanatory, please explain each of them in a bulleted list, the same as you do for the registry parameters just below. It wouldn't hurt to point to where the other device configuration settings are explained in the platform driver configuration (use :ref:`link text <Device-Configuration-File>` as the reference where "link text" is whatever text you want to appear to the reader).